packages/mpg123/mpg123.0.1/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+synopsis: "MP3 decoding library"
+description: "OCaml bindings to libmpg123"
+maintainer: ["Michael Bacarella <m@bacarella.com>"]
+authors: ["Michael Bacarella <m@bacarella.com>"]
+license: "LGPL-2.1-or-later with OCaml-LGPL-linking-exception"
+homepage: "https://github.com/mbacarella/mpg123"
+bug-reports: "https://github.com/mbacarella/mpg123/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ctypes"
+]
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+depexts: [
+  ["libmpg123-dev"] {os-family = "debian"}
+]
```

Comment:
I just opened https://github.com/ocaml/opam-repository/pull/17504 to avoid having to define the depexts manually. I'll update this PR and upstream when this is merged.

Reply:
Oh right, conf- stuff. Thanks!